core/codecArrayBuffer.js

```diff
@@ -19,7 +19,7 @@ sjcl.codec.arrayBuffer = {
   fromBits: function (arr, padding, padding_count) {
     var out, i, ol, tmp, smallest;
     padding = padding==undefined  ? true : padding
-    padding_count = padding_count || 8
+    padding_count = padding_count || 16
```

Comment:
It throws an exception when set to 8, see 
mac[2] ^= data.getUint32(i+8);

in _computeTag